### ISSUE TITLE
Fix "popd: not found" by switching from dash to bash as sh

### ISSUE THREAD

**Issue by MPV** [startersclan/docker-sourceservers#3]
Hi,
Would you be open to change from `dash` to `bash` so `srcds_run` runs without warnings like this?

```shell
/server/srcds_run: 32: /server/srcds_run: pushd: not found
/server/srcds_run: 35: /server/srcds_run: popd: not found
```

For example by doing something like this:

```Dockerfile
RUN cp /bin/bash /bin/sh
```

Or a more sophisticated version:
```Dockerfile
# make /bin/sh symlink to bash instead of dash:
RUN echo "dash dash/sh boolean false" | debconf-set-selections
RUN DEBIAN_FRONTEND=noninteractive dpkg-reconfigure dash
```

https://superuser.com/questions/715722/how-to-do-dpkg-reconfigure-dash-as-bash-automatically/1064247#1064247

**Comment by joeltimothyoh** [comment]
Do you find the errors affecting behavior of the server in any way? Source games tend to spit out a lot errors but that often do not pose issues with functionality. If it is a major issue, perhaps we'll consider adding that into the build Dockerfile.

**Comment by leojonathanoh** [comment]
@MPV We can resolve the `/bin/sh` to `/bin/bash` mapping, since `/bin/sh` has quirks in ubuntu such as `exec` not working as expected. 

Is there any particular reason for to use `srcds_run` wrapper, instead of directly calling the `srcds_linux` binary directly? If we don't use the wrapper script then simply skip over any of its associated quirks and its incompatibility with containerization.

**Comment by MPV** [comment]
Thanks, I tried again now using `srcds_linux` and finally got it working properly. 

I'm setting it up to run using Kubernetes, and when I had tried to send arguments to `srcds_run/linux` using multiple lines which I didn't get to work, but now adding them to the same line I got it to work well.

Closing this, now agreeing with your reasoning and recommendation on `srcds_linux` over `srcds_run`. 

Thanks for taking your time to help and share.
